docs/source/user-manual/links-to-item.md

```diff
@@ -1,10 +1,10 @@
 ---
 myst:
   html_meta:
-    "description": "User manual on how to find all links and relations to the current item."
-    "property=og:description": "User manual on how to find all links and relations to the current item."
-    "property=og:title": "Finding links and relations to the current item."
-    "keywords": "Volto, Plone, frontend, React, User manual, links, relations, references, related content"
+    'description': 'User manual on how to find all links and relations to the current item.'
+    'property=og:description': 'User manual on how to find all links and relations to the current item.'
+    'property=og:title': 'Finding links and relations to the current item.'
+    'keywords': 'Volto, Plone, frontend, React, User manual, links, relations, references, related content'
```

Comment:
Bad editor config, no cookie for you! Please revert above changes.

Reply:
@stevepiercy Sorry, will fix. But it's NOT simply bad editor config. As we've discussed before, I don't know how to make VSCode use double quotes here but single quotes in javascript code samples.